UPGRADE-3.x.md

```diff
@@ -1,6 +1,26 @@
 UPGRADE 3.x
 ===========
 
+## Deprecated accessing to a non existing value when adding field to `showMapper` and `listMapper`.
+
+Before:
+```php
+$showMapper->add('notExistingField');
+$listMapper->add('notExistingField');
+```
+Was displaying nothing in the list and the show views without any warning or error.
+
+But
+```php
+$formMapper->add('notExistingField');
+```
+Was throwing an exeception.
+
+In the next major:
+This will trigger an exception in order if no getter/isser/hasser is found for the property. Since most of
```

Comment:
```suggestion
An exception will be thrown in order if no getter/isser/hasser is found for the property. Since most of
```
What do you mean with "in order"?

Reply:
Nothing, I changed the sentence afterward. Sorry, this update note was written too quickly... It should be better now.